src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -443,11 +439,14 @@ public function close()
 	 */
 	public function __construct(array $options)
 	{
-        if (isset($options['log4php.properties'])) {
-            \Logger::configure($options['log4php.properties']);
-        }
-        $this->_logger = \Logger::getLogger('Pami.ClientImpl');
-	    $this->_host = $options['host'];
+		$this->setLogger(
+			new Log4phpAdapter(
```

Comment:
I agree, and in our own implementation we set the logger using the `$config['logger']` parameter.

How about this: I do the same here, but if the `$config['logger']` property is not set, we keep the behaviour as proposed in this PR?

I think this satisfies both cases: existing users don't have to change anything, and users who want to use PSR-3 can just inject a logger using `$config['logger']`.